src/utils/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin set-flex-property {
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
use more descriptive names for your mixins. I should understand exactly what it does without the need to look for its implementation. But I would add display: flex here as well